addon/mixins/route.js

```diff
@@ -200,6 +200,23 @@ export default Ember.Mixin.create({
   },
 
   /**
+   Call additional functions after finding the infinityModel in the Ember data store.
+   @private
+   @method _afterInfinityModel
+   @param {Function} infinityModelPromise The resolved result of the Ember store find method. Passed in automatically.
+   @return {Ember.RSVP.Promise}
+  */
+  _afterInfinityModel(_this, infinityModelPromiseResult) {
+    if (typeof _this.afterInfinityModel === 'function') {
+      return function(infinityModelPromiseResult) {
+        return _this.afterInfinityModel(infinityModelPromiseResult);
+      };
+    } else {
+      return infinityModelPromiseResult;
```

Comment:
shouldn't this be wrapped in a function? in either case, the return value of this method should be a function that accepts `infinityModelPromiseResult`. In fact you might just do:

```javascript
_afterInfinityModel(_this) {
  return function (infinityModelPromiseResult) {
    if (typeof _this.afterInfinityModel === 'function') {
      return _this.afterInfinityModel(infinityModelPromiseResult);
    } else {
      return infinityModelPromiseResult;
    }
  };
}
```

I'm not seeing a test failure for this though, but I'm p sure `infinityModelPromiseResult` isn't defined when the method is called on line 270